### PR TITLE
[@types/jest] Fix some typos in comments

### PR DESCRIPTION
**Pull request by ghost**
Closes #59960 

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

File: types/jest/index.d.ts [hunk 1]
```diff
@@ -584,7 +584,7 @@ declare namespace jest {
          *
          * - `table`: Tagged Template Literal
          * - `name`: String the title of the test, use `$variable` to inject test data into the test title from the tagged template expressions.
-         * - `fn`: Function the test to be ran, this is the function that will receive the test data object..
+         * - `fn`: Function the test to be run, this is the function that will receive the test data object.
```
Comment (mrazauskas) [inline types/jest/index.d.ts:587]: Line 581 (just above) also has "ran". Could you fix that too, please?

**Comment by mrazauskas** [comment]
Could you add "Closes #59960" to the description?

The link PR is a duplicate. So if your PR would have "closes", merging it would automatically close the linked one. Reference: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests

**Review by mrazauskas — APPROVED** [review]

**Comment by mrazauskas** [comment]
Thanks! Looks good for my eye.

**Review by saboya — APPROVED** [review]

**Review by jakebailey — APPROVED** [review]

**Comment by codershiba** [comment]
Ready to merge

**Pull request merged**